bluesky_widgets/examples/qt_run_tree_view.py

```diff
@@ -0,0 +1,48 @@
+"""
+Select some runs and click the button. Their IDs will be printed to the
```

Comment:
Clearly this needs updating. Just marking it so we don't forget before merging.

Reply:
True, missed this - I will fix it up.